tasks/44_time-simulation/time-simulation.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * class SimTime
+ */
+export class SimTime {
```

Comment:
See description of task 44

function SimTime() {
  // ...
}

let sim = new SimTime();

Reply:
Fixed.